packages/calcite-components/src/components/block/resources.ts

```diff
@@ -42,3 +42,5 @@ export const ICONS = {
   valid: "check-circle",
   invalid: "exclamation-mark-triangle",
 };
+
+export type Position = "start" | "end";
```

Comment:
Can you extract from the shared `Position` interface instead of creating a component-specific one? Like [this](https://github.com/Esri/calcite-design-system/blob/8edeb3645fb8dee31df60165b0f82e13aa290e19/packages/calcite-components/src/components/accordion-item/accordion-item.tsx#L76)?